dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```diff
@@ -212,12 +212,17 @@ jenkins:
                   key: "JENKINS_JAVA_BIN"
                   value: "<%= agent['agentJavaBin'] ? agent['agentJavaBin'] : @jcasc['agents_setup']['ubuntu']['agentJavaBin'] %>"
             <%- if agent['javaHome'] -%>
+            <>
```

Comment:
```suggestion
```
typo?

Reply:
👍 
fixed post-merge in https://github.com/jenkins-infra/jenkins-infra/commit/789d922c9f7396b523b1f2b4b10087f25c46126e